docker-compose.test.yml

```diff
@@ -8,6 +8,8 @@ services:
       - ./testdata/files:/data
     environment:
       UPDATE_WAIT_INTERVAL_MINUTES: 0
+      LOG_SCOPE_AUTHORIZATION: "debug"
+      LOG_SCOPE_SEARCH: "debug"
```

Comment:
do we want to turn these logs off on ACC/PROD?

Reply:
I suppose we do, yes

Reply:
will remove post merge

Reply:
misread, these are only in the test environment. This is fine